src/Paysera/Bundle/PhpGeneratorBundle/Twig/FieldDefinitionExtension.php

```diff
@@ -48,7 +49,7 @@ public function resolveDateTypeFormat(DateTimePropertyDefinition $definition)
                 if ($definition->getFormat() !== null) {
                     return $definition->getFormat();
                 }
-                return \DateTimeInterface::RFC3339;
+                return \DateTime::RFC3339;
```

Comment:
why here interface is replaced with class?

Reply:
Because it's available only with PHP 7.2 - https://travis-ci.org/paysera/util-raml-code-generator/jobs/410714438